Python/pathconfig.c

```diff
@@ -108,7 +108,7 @@ pathconfig_global_init(void)
     _PyInitError err;
     _PyCoreConfig config = _PyCoreConfig_INIT;
 
-    err = _PyCoreConfig_Read(&config);
+    err = _PyCoreConfig_Read(&config, 1);
```

Comment:
Not sure about this one, this is the minimal change to make this code compile with my other changes and without any changes to its behaviour.  I haven't check the code yet to see if the changed behaviour you get with "0" as the second argument would be more useful here.